From bff847d9a0be363573af79a634a4b6b03e17023d Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Wed, 26 Jul 2017 13:27:27 +0800
Subject: [PATCH 1425/5242] MLK-16085 net: fsl: kconfig: add
 NET_VENDOR_FREESCALE dependency with ARM64

commit  2a7e3885f08d5d998ea8fe1168c1847d8b02099b from
https://source.codeaurora.org/external/imx/linux-imx.git

Add NET_VENDOR_FREESCALE dependency with ARCH_MXC_ARM64 for i.MX8x
platforms.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Pete Zhang <pete.zhang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/freescale/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/Kconfig b/drivers/net/ethernet/freescale/Kconfig
index 285d83c..6dc6d79 100644
--- a/drivers/net/ethernet/freescale/Kconfig
+++ b/drivers/net/ethernet/freescale/Kconfig
@@ -8,7 +8,7 @@ config NET_VENDOR_FREESCALE
 	depends on FSL_SOC || QUICC_ENGINE || CPM1 || CPM2 || PPC_MPC512x || \
 		   M523x || M527x || M5272 || M528x || M520x || M532x || \
 		   ARCH_MXC || ARCH_MXS || (PPC_MPC52xx && PPC_BESTCOMM) || \
-		   ARCH_LAYERSCAPE || COMPILE_TEST
+		   ARCH_LAYERSCAPE || ARCH_MXC_ARM64 || COMPILE_TEST
 	---help---
 	  If you have a network (Ethernet) card belonging to this class, say Y.
 
-- 
1.7.9.5

